test/DynamoCoreTests/CustomNodeWorkspaceOpening.cs

```diff
@@ -38,6 +38,21 @@ public void CanOpenWorkspaceWithMissingCustomNodeThenFixByOpeningNeededCustomNod
         }
 
         [Test]
+        public void CanLoadReadOnlyNode()
+        {
+            // Open a read-only custom node
+            OpenTestFile(@"core\CustomNodes", "add_Read_only.dyf");
```

Comment:
how did you mark the file read only? with the file system? can you add a check here that makes sure the custom node file is still marked readonly to make sure the test is actually testing this. (I'm just imagining weird things can happen when this gets moved to the build machine)